src/plugins/intel_gna/legacy/include/legacy/details/ie_cnn_network_tools.h

```diff
@@ -4,16 +4,16 @@
 
 /**
  * @brief A header file for CNNNetwork tools
- * 
+ *
  * @file ie_cnn_network_tools.h
  */
 #pragma once
+#include <legacy/ie_layers.h>
```

Comment:
All OpenVINO headers should be included via `""`

Reply:
Thanks for comment.
This is results of applying of automatic fix jobs for clan-format, this is just reorder of existing includes.
Fix of includes types would be done in separate tasks.